Dockerfile

```diff
@@ -0,0 +1,23 @@
+FROM ubuntu:16.04 as base
```

Comment:
Some concerns:

- We want to keep everything in a virtualenv and not install deps system wide
- We want to avoid running the app itself as root
- Mimicking the prod setup and using uwsgi instead of built-in gunicorn is preferred

You can see the Dockerfile for iris-relay as an example: https://github.com/linkedin/iris-relay/blob/master/Dockerfile